augur/io/sqlite3.py

```diff
@@ -0,0 +1,137 @@
+import os
+import pandas as pd
+import sqlite3
+from typing import List
+
+
+class Sqlite3Database:
+    def __init__(self, file:str=''):
```

Comment:
As mentioned elsewhere, I wish these new classes had docstrings that included details like that the database will be stored in memory when `file` is omitted, etc.

Reply:
Added docstrings: fba5ab299a7625312fe910da458c008dd23aac35

Extracted the in-memory database feature into a separate commit: 9f4034a4f3830dd5563f311204d880780117559c